### PR TITLE
Confirmation dialog for "Jump to test class"

### PR DESCRIPTION
**Pull request by hernanmd**
This PR add a confirmation dialog to avoid command "Jump to test class" in the Calypso Browser to create automatically a class if there is none, as reported in #14533

### CODE REVIEW

File: src/Calypso-SystemPlugins-SUnit-Browser/ClyJumpToTestClassCommand.class.st [hunk 1]
```diff
@@ -50,9 +50,16 @@ ClyJumpToTestClassCommand >> defaultMenuItemName [
...
+	self systemEnvironment 
+		classNamed: (self testClassNameFor: targetClass)
+		ifAbsent: [
+			(self confirm: 'Do you want to create a test class for: ' , targetClass asString )
+				ifFalse: [ ^ self ] ].
+
 	[ browser selectClass: (self testClassFor: targetClass) ]
 		on: ClyInvalidClassForTestClassGeneration
-		do: [ :ex | self inform: 'Cannot generate test class for ' , ex baseClass printString , '.' ]
+		do: [ : ex | self inform: 'Cannot generate test class for ' , ex baseClass printString , '.' ]
```
Comment (Ducasse) [inline src/Calypso-SystemPlugins-SUnit-Browser/ClyJumpToTestClassCommand.class.st]: Can you remove this space? : ex -> :ex

**Comment by Ducasse** [comment]
Broken tests are unrelated (take too much time)

File: src/Calypso-SystemPlugins-SUnit-Browser/ClyJumpToTestClassCommand.class.st (hunk 1 above)
Comment (hernanmd) [reply src/Calypso-SystemPlugins-SUnit-Browser/ClyJumpToTestClassCommand.class.st]: I have never understood this. For method parameters it is accepted to have the argument with spaces but for block parameters it isn't?

Comment (Ducasse) [reply src/Calypso-SystemPlugins-SUnit-Browser/ClyJumpToTestClassCommand.class.st]: Let us not discuss that here. This is not because the syntax allows it that we should have strangely written method. 
99.9 % of the block are written [ :each and not [: each so let us make things simple uniform and easy.
